src/main/java/eu/donals/Location.java

```diff
@@ -33,13 +31,6 @@ public NPC(String name, String about, String item, String riddle, String answer,
         }
     }
 
-    public void printStage() {
-        System.out.println(getName() + "\n" + getDescription() + "\n" + Arrays.toString(getItems()) + "\n" + Arrays.toString(getNeighbours()));
-    }
-
-    // ------- Get and Set methods ------- //
-    public void setName(String x) { this.name = x; }
-
     public String getName() { return name; }
     public String getDescription() { return description; }
 
```

Comment:
A way to solve this naming problem would be for instance to put the getters and setters for NPC in its own class. This makes for longer lines of codes though when getting a name for instance: location.getNpc().getName(). It might be better to just change the name.